### PR TITLE
Broker Ansibleapp

### PR DESCRIPTION
**Pull request by eriknelson**
* ansibleapp assets
* Enahnce asbcli to support deploying broker ansibleapp
* Support configurable openshift target cluster in the broker
* New config exs updated

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -29,3 +29,6 @@ etc/dev.config.yaml
 etc/prod.config.yaml
 
 .python-version
+ansible/roles/ansible-service-broker-openshift/tasks/main_*
+build.sh
+run.sh
```
Comment (fabianvf) [inline .gitignore]: What are these?

File: .gitignore [hunk 2]
```diff
@@ -29,3 +29,6 @@ etc/dev.config.yaml
 etc/prod.config.yaml
 
 .python-version
+ansible/roles/ansible-service-broker-openshift/tasks/main_*
```
Comment (fabianvf) [inline .gitignore]: Can probably ignore any main_.yml files

File: ansible/main.yml [hunk 3]
```diff
@@ -0,0 +1,107 @@
+############################################################
+# TODO: Need to configure things so this etcd pod is linked up
```
Comment (fabianvf) [inline ansible/main.yml]: Does etcd not link up?

File: ansible/main.yml [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+        - device-mapper-devel
+    - name: Create directory layout
+      file: path={{item}} state=directory
+      with_items:
+        - /opt/go/src/github.com/fusor
+        - "{{etc_dest}}"
+        - "{{bin_dest}}"
+    - name: Install golang v1.8
+      shell: curl -L "https://storage.googleapis.com/golang/go1.8.linux-amd64.tar.gz" |
+        tar xz --directory=/usr/local
```
Comment (fabianvf) [inline ansible/main.yml]: At some point should probably use unarchive module for this

File: ansible/main.yml [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+      args:
+        creates: /usr/local/go
+        warn: false
+    - name: Expose go via PATH
+      shell: export PATH=$PATH:/usr/local/go/bin
```
Comment (fabianvf) [inline ansible/main.yml]: Should be able to use a module here too

File: ansible/main.yml [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+        PATH: "{{ansible_env.PATH}}:/usr/local/go/bin"
+    - name: Build ansible-service-broker
+      make:
+        chdir: /opt/go/src/github.com/fusor/ansible-service-broker
+        target: build
+      environment:
+        PATH: "{{ansible_env.PATH}}:/usr/local/go/bin"
+    - name: Install ansible-service-broker
+      shell: cp {{item.src}} {{item.dest}}
+      with_items:
```
Comment (fabianvf) [inline ansible/main.yml]: Module here too

File: ansible/main.yml (hunk 3 above)
Comment (eriknelson) [inline ansible/main.yml]: Links fine, this is an old TODO. Will remove.

File: ansible/main.yml (hunk 4 above)
Comment (eriknelson) [inline ansible/main.yml]: I'll update this.

File: .gitignore (hunk 1 above)
Comment (eriknelson) [inline .gitignore]: Some of my helper scripts.

**Comment by eriknelson** [comment]
@fabianvf I posed the question earlier in #anisbleapp. The more I think about it, it should get moved into ansibleapp-library to use the CI we have in place and to focus that stuff in one place. I'm going to close this PR and pare it down to the work that belongs here, and move the ansibleapp files over to ansibleapp library.

**Pull request closed without merging**